### DISCUSSION TITLE
Feature: A way to search for files

### DISCUSSION THREAD

**Discussion by Felitendo** [diced/zipline#480] [Ideas]
I think this feature should be essential for a file upload/managing platform and I'm really surprised that Zipline still hasn't gotten this feature.. I know you can sort by File Size and other factors, but there is no way to search for specific files like "MyPictureOfAHouse.png"

**Comment by diced** [comment]
Will be added in v4

**Comment by tigattack** [comment]
This would be a huge feature IMO. The current lack of it is _almost_ a deal-breaker for me.

Zipline is super nice to use, nice enough that I can put up with the lack of a search feature, but it would be amazing to have it!

**Comment by diced** [comment]
its been added in v4 
![image](https://github.com/user-attachments/assets/2c3c9699-f784-4a2e-83ef-48ca89f0fe08)
